diff --git a/node_modules/react-native-toast-message/src/index.js b/node_modules/react-native-toast-message/src/index.js
index 3f14bb5..aae57b7 100644
--- a/node_modules/react-native-toast-message/src/index.js
+++ b/node_modules/react-native-toast-message/src/index.js
@@ -11,7 +11,7 @@ import { stylePropType } from './utils/prop-types';
 import { isIOS } from './utils/platform';
 import styles from './styles';
 
-const FRICTION = 8;
+const FRICTION = 1;
 
 const defaultComponentsConfig = {
   // eslint-disable-next-line react/prop-types
@@ -128,7 +128,7 @@ class Toast extends Component {
 
   componentDidMount() {
     const noop = {
-      remove: () => {}
+      remove: () => { }
     };
     this.keyboardDidShowListener = isIOS
       ? Keyboard.addListener('keyboardDidShow', this.keyboardDidShow)
diff --git a/node_modules/react-native-toast-message/src/styles.js b/node_modules/react-native-toast-message/src/styles.js
index 2bc826b..9584c17 100644
--- a/node_modules/react-native-toast-message/src/styles.js
+++ b/node_modules/react-native-toast-message/src/styles.js
@@ -6,7 +6,7 @@ export default StyleSheet.create({
     alignItems: 'center',
     justifyContent: 'center',
     left: 0,
-    right: 0
+    right: 0,
   },
   top: {
     top: 0
